### PR TITLE
leap 1.4.0: Add negative test: year divisible by 200, not divisible by 400

### PR DESCRIPTION
**Pull request by sshine**
Someone thought they could write `year mod 200 = 0`. Test didn't fail.

### CODE REVIEW

File: exercises/leap/canonical-data.json [hunk 1]
```diff
@@ -33,6 +33,14 @@
...
+      "property": "leapYear",
+      "input": {
+        "year": 1800
+      },
+      "expected": false
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (petertseng) [inline exercises/leap/canonical-data.json]: routine objection to unrelated change

Comment (sshine) [reply exercises/leap/canonical-data.json]: Fixed. Thanks.
